Jellyfin.Plugin.Webhook/Destinations/BaseOption.cs

```diff
@@ -97,6 +117,11 @@ public abstract class BaseOption
     /// </summary>
     public Guid[] UserFilter { get; set; } = Array.Empty<Guid>();
 
+    /// <summary>
+    /// Gets or sets the Media-Content type of the webhook body. Should not be set by User except for Generic Client.
+    /// </summary>
+    public WebhookMediaContentType MediaContentType { get; set; } = WebhookMediaContentType.Json;
```

Comment:
This should probably possible to set in the webui

Reply:
I do have it available to set in the Generic Client via a header, though at the moment it only supports json, xml, and generic/plaintext. For the individual pre-built clients though I think it makes sense to keep it hidden, or at least on a per-client basis. Discord webhooks for example only allow for JSON, so it makes no sense to expose an option that invariably breaks functionality if changed.